lib/Config.js

```diff
@@ -755,6 +760,11 @@ class Config extends EventEmitter {
                 this.utapi.expireMetricsTTL = config.utapi.expireMetricsTTL;
             }
         }
+        if (Object.keys(this.locationConstraints).some(
```

Comment:
Maybe it's overkill, but it could be helpful to also assert the `redis` and `localCache` properties.

Reply:
The section that asserts `config.utapi` asserts `redis` and `localcache`, which is why I check for `this.utapi` here. If `this.utapi` exists, `redis` and `localcache` will too.